app/modules/transactions.js

```diff
@@ -100,6 +107,18 @@ export const generateBalanceInfo = (
   })
 }
 
+// This adds some random bits to the transaction to prevent any hash collision.
+const attachAttributesForEmptyTransaction = (config: api.apiConfig) => {
```

Comment:
Without this function (which I dug out of neon-js) there was validation errors for every single invocation transaction but by adding it here I am able to perform offline signing for any transaction type 🎉